rotkehlchen/db/history_events.py

```diff
@@ -375,3 +376,25 @@ def get_value_stats(      # pylint: disable=no-self-use
             except DeserializationError as e:
                 log.debug(f'Failed to deserialize amount {row[1]}. {str(e)}')
         return usd_value, assets_amounts
+
+    def get_extra_data(self, event_identifier: int) -> Optional[dict[str, Any]]:
```

Comment:
```suggestion
    def get_extra_data(self, identifier: int) -> Optional[dict[str, Any]]:
```
Since history events have `identifier` & `event_identifier` fields.